modules/widget-visibility/widget-conditions.php

```diff
@@ -349,7 +352,11 @@ public static function widget_conditions_admin( $widget, $return, $instance ) {
 							</div>
 
 							<div class="condition-control">
-								<span class="condition-conjunction"><?php echo esc_html_x( 'or', 'Shown between widget visibility conditions.', 'jetpack' ); ?></span>
+								<span class="condition-conjunction">
+									<?php
+										echo esc_html_x( 'or', 'Shown between widget visibility conditions.', 'jetpack' );
```

Comment:
This probably shouldn't be broken out onto it's own line from the php opening and closing tags.